### PR TITLE
Farm 53 add exploitation

### PR DESCRIPTION
**Pull request by Erriep**

### CODE REVIEW

**Review by AntoineCAU — DISMISSED** [review]

File: src/components/AddExploitation/index.jsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import { toast } from 'react-toastify';
+import { useHistory } from 'react-router-dom';
+import User from '../../contexts/User';
+import StyledExploitation from './styles';
+
+function AddExploitation() {
+  const history = useHistory();
+  const { user } = useContext(User);
+  const idUser = user.id;
+  const [datas, setDatas] = useState();
```
Comment (LoicBrassart) [inline src/components/AddExploitation/index.jsx]: Always give an initial value, if only to be sure the variable has the right type right off the bat

File: src/components/AddExploitation/index.jsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  }, [datas]);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    axios
+      .post(`${process.env.REACT_APP_API_URL}/exploit`, details)
+      .then((data) => {
+        setDatas(data.data.insertId);
+      });
+  };
```
Comment (LoicBrassart) [inline src/components/AddExploitation/index.jsx]: ... Wait, what ?

File: src/components/MyProfile/index.jsx [hunk 3]
```diff
@@ -26,6 +27,15 @@ function MyProfile() {
     setIsEdit(!isEdit);
   };
 
+  useEffect(() => {
+    axios
+      .get(`${process.env.REACT_APP_API_URL}/exploit/linkuser/${user.id}`)
+      .then(({ data }) => {
+        setExloitUser(data);
+        console.log(exploitUser);
```
Comment (LoicBrassart) [inline src/components/MyProfile/index.jsx]: Nope, still no `console.log`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please be more careful while adding last-minute features to this project

**Review by AntoineCAU — APPROVED** [review]

File: src/components/MyProfile/index.jsx [hunk 4]
```diff
@@ -8,9 +8,10 @@ function MyProfile() {
   const [infos, setInfos] = useState([]);
   const { user } = useContext(User);
   const [isEdit, setIsEdit] = useState(false);
+  const [exploitUser, setExloitUser] = useState([]);
```
Comment (LoicBrassart) [inline src/components/MyProfile/index.jsx:11]: Typo in the function's name

**Review by LoicBrassart — APPROVED** [review]
One minor typo

**Pull request merged**